inference-engine/tests/functional/inference_engine/ir_serialization/partial_shape_deserialization.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <inference_engine.hpp>
+#include <memory>
+#include <string>
+#include "manager.hpp"
+
+#include "graph_comparator.hpp"
+#include "ie_blob.h"
+#include "ngraph/op/parameter.hpp"
+#include "ngraph/type/element_type.hpp"
+#include "ngraph/variant.hpp"
+#include "openvino/core/preprocess/input_tensor_info.hpp"
+#include "openvino/runtime/core.hpp"
+
+class PartialShapeDeserialization : public testing::Test {
```

Comment:
Please extend visitor tests to cover changes in PartialShape

Reply:
Done.